tests/unit/map-test.js

```diff
@@ -14,6 +14,7 @@ module('TrackedMap', function(hooks) {
 
     assert.equal(map.get('foo'), 123);
     assert.equal(map.size, 1);
+    assert.ok(map instanceof Map);
```

Comment:
Something fishy I've noticed while trying to make sense of things...

```js
  test('constructor', assert => {
    Object.setPrototypeOf(TrackedMap.prototype, Map.prototype);
    let map = new TrackedMap([['foo', 123]]);

    assert.equal(map.get('foo'), 123);
    assert.equal(map.size, 1);
    assert.ok(map instanceof Map);
  });
```
passes.

_but_, remove the `setPrototypeOf`, and it fails. _even though_, `map.ts`  also declares
```js
Object.setPrototypeOf(TrackedMap.prototype, Map.prototype);
```

Reply:
oh! I needed to run `yarn build:ts` :thinking: 

ok, fixes for everything incoming :D

Reply:
nope, I'm just more confused now :-\

Reply:
@NullVoxPopuli did you get this sorted out for your own understanding at minimum? 😅